scripts/circle-ci/run-circle-tests.sh

```diff
@@ -52,7 +52,11 @@ JAVA_GC_LOGGING_OPTIONS="${JAVA_GC_LOGGING_OPTIONS} -Xloggc:build-%t-%p.gc.log"
 # External builds have a 4GB limit so we have to tune everything so it fits in memory (only just!)
 if [[ $INTERNAL_BUILD == true ]]; then
     BASE_GRADLE_ARGS+=" --parallel"
-    export _JAVA_OPTIONS="-Xmx1024m ${JAVA_GC_LOGGING_OPTIONS}"
+    if [[ $CIRCLE_NODE_INDEX == 7 ]]; then
```

Comment:
Not an expert on this so very likely wrong: think integer comparison is different

Reply:
http://tldp.org/LDP/abs/html/comparison-ops.html

Reply:
Thanks for the link - will fix this up when I can.

The first attempt definitely didn't work, because the errorprone job OOMed!